target/linux/sunxi/image/cortex-a7.mk

```diff
@@ -63,13 +63,14 @@ define Device/lemaker_bananapi
 endef
 TARGET_DEVICES += lemaker_bananapi
 
-define Device/lemaker_bananapi-m2-ultra
-  DEVICE_VENDOR := LeMaker
+define Device/sinovoip_bananapi-m2-ultra
+  DEVICE_VENDOR := Sinovoip
   DEVICE_MODEL := Banana Pi M2 Ultra
-  DEVICE_PACKAGES:=kmod-rtc-sunxi kmod-ata-sunxi
+  DEVICE_PACKAGES := kmod-rtc-sunxi kmod-ata-sunxi kmod-brcmfmac \
```

Comment:
Please don't introduce spaces next to the equal sign, as this does not match the style of the remaining entries.